js/imageslider/Index.svelte

```diff
@@ -0,0 +1,210 @@
+<svelte:options accessors={true} />
+
+<script lang="ts">
+	import type { Gradio, SelectData, ValueData } from "@gradio/utils";
+	import StaticImage from "./shared/SliderPreview.svelte";
+	import ImageUploader from "./shared/SliderUpload.svelte";
+	import { afterUpdate } from "svelte";
+
+	import { Block, Empty, UploadText } from "@gradio/atoms";
+	import { Image } from "@gradio/icons";
+	import { StatusTracker } from "@gradio/statustracker";
+	import { upload, type FileData } from "@gradio/client";
+	import type { LoadingStatus } from "@gradio/statustracker";
+
+	type sources = "upload" | "webcam" | "clipboard" | null;
+
+	let stream_state = "closed";
+	let _modify_stream: (state: "open" | "closed" | "waiting") => void = () => {};
```

Comment:
Don't need the modify_stream stuff since this component does not support streaming